.github/workflows/sast.yml

```
@@ -37,10 +37,10 @@ jobs:
     - name: Initialize CodeQL
       uses: github/codeql-action/init@v2
 
-    - name: Setup JDK 8
+    - name: Setup JDK 17
       uses: actions/setup-java@v2
```

actionlint

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)